libs/sdk-ui-filters/src/AttributeFilter@next/Components/Addons/AttributeDisplayFormSelect/AttributeDisplayFormDropdownButton.tsx

```diff
@@ -0,0 +1,52 @@
+// (C) 2021-2022 GoodData Corporation
+
+import React from "react";
+import classNames from "classnames";
+import { WrappedComponentProps, injectIntl } from "react-intl";
+import { Button } from "@gooddata/sdk-ui-kit";
+
+/**
+ * @internal
+ */
+export interface IToggleButtonProps {
+    text: string;
+    isOpened: boolean;
+    toggleDropdown: (e: React.SyntheticEvent) => void;
+}
+
+/**
+ * @internal
+ */
+export type ToggleButtonProps = IToggleButtonProps & WrappedComponentProps;
```

Comment:
Naming seems a bit vague, what about `AttributeDisplayFormDropdownButton` + `IAttributeDisplayFormDropdownButtonProps` so we have it aligned with other components?

Reply:
done